cli/asc.js

```diff
@@ -303,7 +303,7 @@ exports.main = function main(argv, options, callback) {
     // Append entries
     if (asconfig.entries) {
       for (let entry of asconfig.entries) {
-        argv.push(optionsUtil.resolvePath(entry, asconfigDir));
+        argv.push(path.relative(baseDir, optionsUtil.resolvePath(entry, asconfigDir)));
```

Comment:
Given `optionsUtil.resolvePath` https://github.com/AssemblyScript/assemblyscript/blob/348c42b7466054a793fde80a8100eaf9623fff36/cli/util/options.js#L241-L247

I wonder why this is necessary given that `asconfigDir` is *supposed* to be relative to `baseDir` (or absolute) already. If `entry` is relative, we should get `path.join(asconfigDir, entry)` as the result with `asconfigDir = path.join(baseDir, somePath)` (or absolute). As such I'm puzzled how this fixes the problem and would suspect that the problem might be deeper?

Reply:
I'm not sure.  Since I don't have windows, how it handles paths is weird to me.  In other node projects I've found that using all unix paths works well since node handles the conversion.